### ISSUE TITLE
Creating a menu breaks /news

### ISSUE THREAD

**Issue by jslang** [UCF/Community-Relations-WP-Theme#1]
In an attempt to make the site navigation editable via the wordpress admin, I edited footer.php to call wp_nav_menu and created a menu which will display for that call.

For some reason unknown to me, creating this menu breaks the functionality originally displayed when going to the url /news.  Instead of displaying a news post, a 404 page is displayed.  Deleting the menu fixes it.

**Comment by jslang** [comment]
Found out why this is happening.  When we created a menu, we also added a custom link that points to /news that we labeled "News".

When attempting to visit a url that does not exist, wordpress will attempt to best guess what URL you were trying to access.  So when accessing the url /news, wordpress guesses that you meant to go to the first post it finds slugged news-test-post-1.  This works when the custom menu item doesn't exist.  The reason it breaks when it does exist is that whatever label is inputted for the custom link is slugged, so in our case it was slugged "news."  Wordpress assumes this is the url you were attempting to access when trying to access "news", and directs you to that slug which points to the page /news, which does not actually exist.  Thus a 404 is displayed.

An ideal solution to this problem would be to ensure that the category post listing pages function properly, and to link to those ie. "/category/news".  As it stands, even when the theme functions, wordpress is redirecting you to the first post news post to ever exist, which would not result in latest news ever being displayed for the url "news".  Fixing the category post listing page would allow us to link to a chronological listing of news posts.
